bands/site_sections/band_admin.py

```diff
@@ -56,10 +56,10 @@ def everything(self, out, session):
             out.writerow([
                 band.group.name, band.email,
                 band.payment, band.vehicles, band.estimated_loadin_minutes, band.estimated_performance_minutes,
-                band.poc_phone, band.performer_count, band.bringing_vehicle, band.vehicle_info, band.arrival_time,
-                band.bio, band.website, band.facebook, band.twitter, band.other_social_media, band.bio_pic_url,
-                band.wants_panel, band.panel_name, band.panel_length, band.panel_desc, ' / '.join(band.panel_tech_needs_labels),
-                band.w9_url, band.stage_plot_url,
-                band.merch_label if band.merch else '',
-                band.charity_label if band.charity else '', band.charity_donation
+                band.info.poc_phone, band.info.performer_count, band.info.bringing_vehicle, band.info.vehicle_info, band.info.arrival_time,
+                band.bio.desc, band.bio.website, band.bio.facebook, band.bio.twitter, band.bio.other_social_media, band.bio.pic_url,
+                band.panel.wants_panel, band.panel.name, band.panel.length, band.panel.desc, ' / '.join(band.panel.panel_tech_needs_labels),
+                band.taxes.w9_url, band.stage_plot.url,
+                band.merch.merch_label if band.merch else '',
+                band.charity.charity_label if band.charity else '', band.charity.desc
```

Comment:
The problem with this is that it's going to raise an exception when we have any band which hasn't filled out everything.  For this to really work, we'll need to replace e.g. ``band.info.poc_phone`` with something equivalent to ``getattr(band.info, 'poc_phone', '')``.

For now we could probably just do the above.  In theory we could write a custom ``__getattr__`` which would be smarter and allow us to say something like ``band._info_poc_phone`` and automatically translate that to ``band.info.poc_phone`` if ``band.info`` existed and otherwise return an empty string.  But for now just calling ``getattr()`` is probably fine and only marginally more painful than what we have now.